src/Parser/AfflictionWarlock/Constants.js

```diff
@@ -0,0 +1,9 @@
+import SPELLS from 'common/SPELLS';
+
+export const UNSTABLE_AFFLICTION_DEBUFF_IDS = [
+  SPELLS.UNSTABLE_AFFLICTION_DEBUFF_1.id,
```

Comment:
Unstable Affliction works in the way that you can add up to 5 different applications of it on the target (used to be that they stacked the damage, I think on 7.1 or so, they changed it so casting it again doesn't refresh but instead add a new debuff). I'll add a comment in next commit.